src/Lasso.jl

```diff
@@ -301,10 +322,191 @@ function StatsBase.fit{T<:AbstractFloat,V<:FPVector}(::Type{LassoPath},
 
     # Fit path
     path = LassoPath{typeof(model),T}(model, nulldev, nullb0, λ, autoλ, Xnorm)
-    dofit && fit!(path; irls_tol=irls_tol, fitargs...)
+    if dofit
+        fit!(path; irls_tol=irls_tol, fitargs...)
+    else
+        path.λ = zeros(T, 0)
+        path.pct_dev = zeros(T, 0)
+        path.coefs = spzeros(T, p, 0)
+        path.b0 = zeros(T, 0)
+        path.niter = 0
+    end
```

Comment:
You could check `isdefined(path, :coefs)`, or else add a new field that specifies whether or not it has been fit.